blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpRollupsQueryHandlerIntegrationTest.java

```diff
@@ -84,7 +84,7 @@ public void testSingleplotQueryWithEnum() throws Exception {
         final String metric_name = "enum_metric_test" + postfix;
 
         // post multi metrics for ingestion and verify
-        HttpResponse response = postMetric(tenant_id, postAggregatedPath, "sample_enums_payload.json", now, postfix );
+        HttpResponse response = postMetric(tenant_id, postAggregatedPath, "sample_single_enum_payload.json", now, postfix );
```

Comment:
I know this test has been breaking intermittently. But curious, why did you change it to just post one Enum? What's wrong with posting multiple Enums?